diff --git a/node_modules/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp b/node_modules/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp
index 1e2420d..db2e6c1 100644
--- a/node_modules/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp
+++ b/node_modules/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp
@@ -29,12 +29,7 @@
 
 #include <DefaultComponentsRegistry.h>
 #include <DefaultTurboModuleManagerDelegate.h>
-#if __has_include("<autolinking.h>")
-#define AUTOLINKING_AVAILABLE 1
 #include <autolinking.h>
-#else
-#define AUTOLINKING_AVAILABLE 0
-#endif
 #include <fbjni/fbjni.h>
 #include <react/renderer/componentregistry/ComponentDescriptorProviderRegistry.h>
 #include <rncore.h>
@@ -61,10 +56,8 @@ void registerComponents(
   REACT_NATIVE_APP_COMPONENT_REGISTRATION(registry);
 #endif
 
-#if AUTOLINKING_AVAILABLE
   // And we fallback to the components autolinked
   autolinking_registerProviders(registry);
-#endif
 }
 
 std::shared_ptr<TurboModule> cxxModuleProvider(
@@ -78,12 +71,8 @@ std::shared_ptr<TurboModule> cxxModuleProvider(
   //   return std::make_shared<NativeCxxModuleExample>(jsInvoker);
   // }
 
-#if AUTOLINKING_AVAILABLE
   // And we fallback to the CXX module providers autolinked
   return autolinking_cxxModuleProvider(name, jsInvoker);
-#endif
-
-  return nullptr;
 }
 
 std::shared_ptr<TurboModule> javaModuleProvider(
@@ -112,12 +101,10 @@ std::shared_ptr<TurboModule> javaModuleProvider(
     return module;
   }
 
-#if AUTOLINKING_AVAILABLE
   // And we fallback to the module providers autolinked
   if (auto module = autolinking_ModuleProvider(name, params)) {
     return module;
   }
-#endif
 
   return nullptr;
 }
